Samples/Android/Speech Devices SDK Starter App/example/attendee.properties

```diff
@@ -0,0 +1,8 @@
+#ATTENDEELIST=<User_Name@User_signature>
+#for example: ATTENDEELIST=<Jessica@{\
+#				"Version": 0,\
+#				"Tag": "8pzDubBjBxeReeeeEEEEE4iiiiiiiiiiOIgO4LNVPw=",\
+#				"Data": "K0bo9xkUxi234asdfdfeddeekljiklia4TlGMLV76e+6mg2Yr8Ky0VZ9AT4U/kV74awerdeerfeeeeNkxJOedeeepImrYmF/pQ3Nwp4dP2NPphpAaVIA+uuuuuuuuu5G+oZUVdefcj/5IWRLOTcQ31KM8pF4xZkRQNpBMyQFRSP/LGugvBtEweiweD3khQt2aBzIoFeI/24DaaxJ/xNb5FWMAdq6/VtDyxhiAdr3V0X1OknwhG8czUFNkSwkm85lHjukqbACNKDKWVRzv+zUZOS2jicVJh6TCTeWaEMYpg96tBJz348fIpEguZEhvA3KSU+FKu9ZkDQPi/6dw8TH0m+dOV+/qchNiksVlqTUYNMNAj5ZaPWKbAsvhcMjQFp2PEO9Qypk22YhyNkv6wYA72f4/E+B7ApykkLptJziMhwxBK9QIvmbWLa5O2v2H1xQ/oDpl6+yMK+xHDIEzY393rtwI/aEseKE4w7ubXYvhQ0xJR6Tx/NKfrqsuzlABHCbbVdio5PtQpI9C4lDPz7ufAxk8WiaPQ8Bf/mul2eppn1E9cnAaLBS1Frma/JX4nse8LzDk+9xbqfeYJAlZgdK12szrQGqg6o9beeJyeJGtYWguGrCo+sl1DTiTuI7j0RSV2p+FxFLIcauugLYh3J0UYxhe4w4"\
```

Comment:
This voice signature works only with the person who have enrolled this and used that key. I think we should remove this from sample and give either wave file and code for enrollment or document this so that end users can do the enrollment by themselves

Reply:
thanks, I removed the fake signature